scripts/safari.js

```diff
@@ -1807,7 +1813,7 @@ function Safari() {
         }
         var leader = parseInt(player.party[0], 10);
         var species = pokeInfo.species(leader);
-        var dailyBonus = dailyBoost.pokemon == species && !isMega(leader) ? dailyBoost.bonus : 1;
+        var dailyBonus = dailyBoost.pokemon == species && !isMega(leader) && sys.pokemon(leader) != "Floette-EF" ? dailyBoost.bonus : 1;
```

Comment:
Expected '===' and instead saw '=='.<br>Expected '!==' and instead saw '!='.<br>Too many errors. (20% scanned).